.gitignore

```diff
@@ -7,3 +7,7 @@ pymdb
 *.png
 *.pyc
 .DS_Store
+.idea/
+pymdb.egg-info/
+infos.txt
+test_files/A Trip to the Moon 1902/images/
```

Comment:
Not necessary to add this, and the movie file is included in your commit.

Reply:
OK, I think that's fixed now.